source/documentation/index.html.md.erb

```diff
@@ -0,0 +1,59 @@
+---
+title: Documentation
+introduction: >
+  Sass is a stylesheet language that’s compiled to CSS. It allows you to use
+  [variables](/documentation/variables), [nested
+  rules](/documentation/style-rules#nesting),
+  [mixins](/documentation/at-rules/mixin),
+  [functions](/documentation/modules), and more, all with a fully
+  CSS-compatible syntax. Sass helps keep large stylesheets well-organized and
+  makes it easy to share design within and across projects.
+overview: true
+---
+
+* If you're looking for an introduction to Sass, check out [the
+  tutorial](/guide).
+
+* If you want to look up a built-in Sass function, look no further than [the
+  built-in module reference](/documentation/modules).
+
+* If you're calling Sass from JavaScript, you may want the [JS API
+  documentation][js].
+
+* Or the [Dart API documentation][dart] if you're calling it from Dart.
+
+* Otherwise, use the table of contents for the language reference!
+
+[js]: https://github.com/sass/node-sass#usage
+[dart]: https://pub.dartlang.org/documentation/sass/latest/sass/sass-library.html
+
+## Older Versions
+
+This documentation is written for the most recent version of the Sass language.
+If you're using [Dart Sass] <%= impl_version(:dart) %>, you'll have access to
+all the features described here. But if you're using an older version of Dart
+Sass or a deprecated Sass implementation like [LibSass] or [Ruby Sass], there
+may be some behavioral differences.
+
+[Dart Sass]: /dart-sass
+[LibSass]: /libsass
+[Ruby Sass]: /ruby-sass
+
+Anywhere behavior differs between versions or implementations, the documentation
+includes a compatibility indicator like this:
+
+<%= impl_status feature: 'Feature Name', dart: true, libsass: '3.6.0', ruby: false %>
+
+Implementations with a "✓" fully support the feature in question, and
+implementations with a "✗" don't support it all. Implementations with a version
+number started supporting the feature in question at that version.
+Implementations can also be marked as "partial":
+
+<% impl_status dart: true, libsass: :partial, ruby: false do %>
+  Additional details go here.
+<% end %>
+
+This indicates that the implementation only supports some aspects of the
+feature. These compatibility indicators (and many others) have a "▶" button,
+which can be clicked to show more details about exactly how the implementations
+differ and which versions support which aspects of the feature in question.
```

Comment:
IMO this is a common enough UX pattern that we don't need to describe it.
```suggestion
This indicates that the implementation only supports some aspects of the
feature.
```

Reply:
In practice, people have missed this in the past, so I think it's worth mentioning explicitly.